_gitbook/syntax_and_semantics/macros/macro_methods.md

```diff
@@ -20,3 +20,19 @@ person.instance_vars_names #=> ["name", "age"]
 
 Note that in the case of macro defs you need to specify the return type.
 
+Arguments in macro defs are evaluated in running time, not a compile time:
```

Comment:
That's right.  My intended is  "Arguments passed to macro defs method call are not passing macro expansion, but passing to expanded method".